script/import_scripts/smf2.rb

```diff
@@ -250,6 +250,23 @@ def import_attachment(post, attachment)
   end
 
   def postprocess_posts
+    puts '', 'converting bbcode'
+
+    current = 0
+    max = Post.count
+
+    Post.find_each do |post|
+      begin
+        new_raw = convert_bbcode(post.raw.dup)
```

Comment:
Shouldn't this be done during the import? (ie. in `import_posts`)

Reply:
Ideally, yes. I was having trouble getting that to directly work while running the import. I'm not sure if that was due to encoding errors that weren't getting thrown or not. This PR is an effort to try to consolidate my improvements after the fact because they got to be a bit of a mess.

Reply:
I added this in to happen after quote conversion and in the postprocess section. You can change a variable to handle quotes later and mark those posts for rebaking. The reason for doing this is the bbcode conversion would need to happen after quote conversion to make sure all the quoted elements are handled appropriately.

Reply:
Right, quotes often have to be processed afterwards.